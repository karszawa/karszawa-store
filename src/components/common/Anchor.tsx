import styled from "styled-components";

export const A = styled.a`
  display: flex;
  align-items: center;

  &:hover {
    color: #fac75a;
    opacity: 0.7;
    transition: all 0.2s ease-out;
    cursor: pointer;
  }
`;

export const ResetA = styled.a`
  text-decoration: none;
  color: #333;

  &:visited {
    color: #333;
  }
`;
