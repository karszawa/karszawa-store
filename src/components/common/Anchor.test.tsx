import React from "react";
import { A } from "./Anchor";
import { render } from "@testing-library/react";
import "jest-dom/extend-expect";

test("Render Anchor", () => {
  const { asFragment } = render(<A href="https://mercari.com/jp">top</A>);

  expect(asFragment()).toMatchSnapshot();
});
