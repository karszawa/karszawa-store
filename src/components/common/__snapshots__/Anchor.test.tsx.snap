// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Anchor 1`] = `
<DocumentFragment>
  <a
    class="Anchor__A-xh1dxk-0 kwIQeY"
    href="https://mercari.com/jp"
  >
    top
  </a>
</DocumentFragment>
`;
